packages/api/package.json

```diff
@@ -11,7 +11,7 @@
     "test": "jest",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
-    "generate": "prisma migrate dev"
+    "generate": "prisma db push"
```

Comment:
What is the difference between the previous command and this one?

Reply:
Db push is used for schema prototyping and does not create history of changes, yet I think that we should still use prisma migrate each time we are done with modifying the schema.